lib/AST/InlinableText.cpp

```diff
@@ -109,6 +110,15 @@ struct ExtractInactiveRanges : public ASTWalker {
     return false;
   }
 
+  std::pair<bool, Expr *> walkToExprPre(Expr *e) {
```

Comment:
Can you add a comment explaining the CollectionExpr IfConfig stuff here?

Reply:
Sure thing. I’m looking at why recursive conditionals don’t work at the moment. Any ideas?

Reply:
What output are you getting with nested conditionals? It may be a bug in the existing code for stripping #if